docker-compose.yml

```diff
@@ -1,4 +1,4 @@
-version: "3"
+version: '3'
```

Comment:
Is your editor flipping double quotes to single?  Not a big deal but might be noisy if different peoples editors keep flipping files like this back and forth?  We don't have any linting or autoformatting on docker related files to keep these in sync so might be worth deciding on a consistent format and just shepherding it manually?